app/src/main/java/de/westnordost/streetcomplete/quests/diet_type/AddGlutenFree.kt

```diff
@@ -0,0 +1,50 @@
+package de.westnordost.streetcomplete.quests.diet_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.osm.isPlaceOrDisusedPlace
+import de.westnordost.streetcomplete.osm.updateWithCheckDate
+
+
+class AddGlutenFree : OsmFilterQuestType<DietAvailabilityAnswer>() {
+
+    override val elementFilter = """
+        nodes, ways with
+        (
+          amenity ~ restaurant|cafe|fast_food|food_court and food != no
+          or amenity ~ pub|nightclub|biergarten|bar and food = yes
+          or shop ~ supermarket|convenience|deli
```

Comment:
if #6200 gets merged, we should probably also apply the same changes here:

```suggestion
          or shop ~ supermarket|convenience|deli
          or tourism ~ alpine_hut and food != no
```